src/main/kotlin/no/nav/familie/klage/behandling/BehandlingService.kt

```diff
@@ -122,24 +122,32 @@ class BehandlingService(
         behandlingId: UUID,
         påklagetVedtakDto: PåklagetVedtakDto
     ): PåklagetVedtakDetaljer? {
-        if (påklagetVedtakDto.påklagetVedtakstype == PåklagetVedtakstype.INFOTRYGD_TILBAKEKREVING) {
-            return tilPåklagetVedtakDetaljerForInfotrygdVedtak(påklagetVedtakDto)
+        if (påklagetVedtakDto.påklagetVedtakstype == PåklagetVedtakstype.INFOTRYGD_TILBAKEKREVING || påklagetVedtakDto.påklagetVedtakstype == PåklagetVedtakstype.UTESTENGELSE) {
+            return tilPåklagetVedtakDetaljerMedManuellDato(påklagetVedtakDto)
         }
         return påklagetVedtakDto.eksternFagsystemBehandlingId?.let {
             fagsystemVedtakService.hentFagsystemVedtakForPåklagetBehandlingId(behandlingId, it)
                 .tilPåklagetVedtakDetaljer()
         }
     }
 
-    private fun tilPåklagetVedtakDetaljerForInfotrygdVedtak(påklagetVedtakDto: PåklagetVedtakDto) =
+    private fun tilPåklagetVedtakDetaljerMedManuellDato(påklagetVedtakDto: PåklagetVedtakDto) =
         PåklagetVedtakDetaljer(
-            fagsystemType = FagsystemType.TILBAKEKREVING,
+            fagsystemType = utledFagsystemType(påklagetVedtakDto),
             eksternFagsystemBehandlingId = null,
             behandlingstype = "",
             resultat = "",
-            vedtakstidspunkt = påklagetVedtakDto.vedtaksdatoInfotrygd?.atStartOfDay() ?: error("Mangler vedtaksdato")
+            vedtakstidspunkt = påklagetVedtakDto.utledManuellVedtaksdato()?.atStartOfDay() ?: error("Mangler vedtaksdato")
         )
 
+    private fun utledFagsystemType(påklagetVedtakDto: PåklagetVedtakDto): FagsystemType {
+        return when (påklagetVedtakDto.påklagetVedtakstype) {
+            PåklagetVedtakstype.INFOTRYGD_TILBAKEKREVING -> FagsystemType.TILBAKEKREVING
+            PåklagetVedtakstype.UTESTENGELSE -> FagsystemType.UTESTENGELSE
+            else -> error("Kan ikke utlede fagsystemType for påklagetVedtakType ${påklagetVedtakDto.påklagetVedtakstype.name}")
```

Comment:
Når det er en enum så trenger man egentlige ikke å bruke `.name` då det automatiskt blir navnet på enumen :) 
```suggestion
            else -> error("Kan ikke utlede fagsystemType for påklagetVedtakType ${påklagetVedtakDto.påklagetVedtakstype}")
```